core/metrics/src/main/scala-2/org/typelevel/otel4s/metrics/HistogramMacro.scala

```diff
@@ -60,6 +72,31 @@ private[otel4s] trait HistogramMacro[F[_], A] {
   ): Resource[F, Unit] =
     macro HistogramMacro.recordDuration
 
+  /** Records duration of the given effect.
+    *
+    * @example
+    *   {{{
+    * val histogram: Histogram[F] = ???
+    * val attributeKey = AttributeKey.string("query_name")
+    *
+    * def findUser(name: String) =
+    *   histogram.recordDuration(TimeUnit.MILLISECONDS, Attribute(attributeKey, "find_user")).use { _ =>
```

Comment:
```suggestion
    *   histogram.recordDuration(TimeUnit.MILLISECONDS, List(Attribute(attributeKey, "find_user"))).use { _ =>
```

Reply:
aaa, good catch! or I might wrap it in `Attributes` just for contrast